### PR TITLE
chore: Updated the "Getting started" codelab

### PR DESCRIPTION
**Pull request by albertodev01**
_Description of what this PR is changing or adding, and why:_

The [Get started](https://docs.flutter.dev/get-started/codelab) example has a few issues:

  - Some minor fixes are required (like missing `const` constructor)
  - This example uses helper methods instead of widgets. The Flutter team itself recommends to **not** use functions to return widgets on [YouTube](https://www.youtube.com/watch?v=IOyq-eTRhvo) and in the official [documentation](https://github.com/flutter/flutter/pull/95407) (the PR got merged and it will land soon.

I have also submitted a PR [here](https://github.com/flutter/codelabs/pull/265) to update the step-4 so that functions to return widgets aren't used anymore.

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/get-started/codelab.md [hunk 1]
```diff
@@ -380,6 +380,8 @@ a child inside the existing `MyApp` stateless widget.
 
   ```dart
 class RandomWords extends StatefulWidget {
+  const RandomWords();
```
Comment (domesticmouse) [inline src/get-started/codelab.md:383]: I'd prefer the full `const RandomWords({Key? key): super(key:key);` incantation here.

Comment (albertodev01) [reply src/get-started/codelab.md]: Done!

**Comment by domesticmouse** [comment]
According to CI, the snippets are out of date. The source of truth for this Codelab is the `codelabs` git submodule.

I'm guessing you are basing this https://github.com/flutter/codelabs/pull/265?

How about I land that PR, then you can re-base this PR off the updated sub module to make CI happy.

**Comment by domesticmouse** [comment]
You'll need to update the the codelabs sub repo and then update the code snippets. See https://github.com/flutter/website/pull/6721 for step 1 of this process.

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
This PR LGTM, once the code is synced up. Have we lost you, @albertodev01?

**Comment by albertodev01** [comment]
@sfshaza2 I will ping you next week 😄 I'm not feeling well due to covid but I got these PRs in my queue so I won't forget

**Comment by sfshaza2** [comment]
Feel better, @albertodev01!

**Pull request closed without merging**